admin_manual/configuration_server/performance_tuning/webserver_tips.rst

```diff
@@ -12,6 +12,15 @@ on/off``. If it is on, then add this line to ``php.ini`` to turn it off::
 
  safe_mode = Off
 
+Raise max_file_uploads of PHP
+-----------------------------
+
+The PHP setting ``max_file_uploads`` within the ``php.ini`` defaults to ``20``
+on most environments which allows that number of simultaneously uploads.
```

Comment:
Can or should? :) That just was copied from https://secure.php.net/manual/en/ini.core.php#ini.max-file-uploads